spec/models/ingredient_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+describe Ingredient do
+  let!(:veg_ingredient) { FactoryBot.create(:ingredient, :veg) }
+  let!(:non_veg_ingredient) { FactoryBot.create(:ingredient, :non_veg) }
+
+  it "should have valid categories constant" do
```

Comment:
we don't write it like this, change it to

"is expected to have valid categories constant"